src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1487,17 +1495,12 @@ private void OnTransactionDispose(TransactionOperationContext context, long inde
             context.Transaction.InnerTransaction.LowLevelTransaction.OnDispose += transaction =>
             {
                 if (transaction is LowLevelTransaction llt && llt.Committed)
-                    TaskExecutor.Execute(_ =>
-                    {
-                        try
-                        {
-                            _rachisLogIndexNotifications.NotifyListenersAbout(index, null);
-                        }
-                        catch (Exception e)
-                        {
-                            _rachisLogIndexNotifications.NotifyListenersAbout(index, e);
-                        }
-                    }, null);
+                {
```

Comment:
Removing the `TaskExecutor.Execute` do make sense, since there is no actual work here.
But why removing the try/catch ?

Reply:
Nothing can throw there, except OperationCanceledException. Fixed.

Reply:
But we do want to cancel the task, no?

Reply:
Done